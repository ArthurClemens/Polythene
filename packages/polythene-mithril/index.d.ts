export { Button } from "polythene-mithril-button";
export { ButtonGroup } from "polythene-mithril-button-group";
export { Card } from "polythene-mithril-card";
export { Checkbox } from "polythene-mithril-checkbox";
export { Dialog } from "polythene-mithril-dialog";
export { DialogPane } from "polythene-mithril-dialog-pane";
export { Drawer } from "polythene-mithril-drawer";
export { FAB } from "polythene-mithril-fab";
export { Icon } from "polythene-mithril-icon";
export { IconButton } from "polythene-mithril-icon-button";
export { IOSSpinner } from "polythene-mithril-ios-spinner";
export { List } from "polythene-mithril-list";
export { ListTile } from "polythene-mithril-list-tile";
export { MaterialDesignProgressSpinner } from "polythene-mithril-material-design-progress-spinner";
export { MaterialDesignSpinner } from "polythene-mithril-material-design-spinner";
export { Menu } from "polythene-mithril-menu";
export { RadioButton } from "polythene-mithril-radio-button";
export { RadioGroup } from "polythene-mithril-radio-group";
export { Shadow } from "polythene-mithril-shadow";
export { SVG } from "polythene-mithril-svg";
export { Switch } from "polythene-mithril-switch";
