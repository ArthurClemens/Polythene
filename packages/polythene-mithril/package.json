{
  "name": "polythene-mithril",
  "version": "1.4.2",
  "description": "",
  "main": "dist/polythene-mithril",
  "module": "dist/polythene-mithril.mjs",
  "scripts": {
    "clean": "rm -rf dist/*",
    "rollup": "../../node_modules/rollup/bin/rollup -c ../../scripts/bundling/rollup.umd.js && ../../node_modules/rollup/bin/rollup -c ../../scripts/bundling/rollup.es.js",
    "build-standalone": "ENTRY=scripts/standalone.js DEST=./dist/polythene-mithril-standalone ../../node_modules/rollup/bin/rollup -c ../../scripts/bundling/rollup.browser.js",
    "build": "npm run clean && npm run rollup && npm run build-standalone"
  },
  "files": [
    "dist",
    "README.md"
  ],
  "dependencies": {
    "polythene-css": "^1.4.2",
    "polythene-mithril-base": "^1.4.2",
    "polythene-mithril-button": "^1.4.2",
    "polythene-mithril-button-group": "^1.4.2",
    "polythene-mithril-card": "^1.4.2",
    "polythene-mithril-checkbox": "^1.4.2",
    "polythene-mithril-dialog": "^1.4.2",
    "polythene-mithril-dialog-pane": "^1.4.2",
    "polythene-mithril-drawer": "^1.4.2",
    "polythene-mithril-fab": "^1.4.2",
    "polythene-mithril-icon": "^1.4.2",
    "polythene-mithril-icon-button": "^1.4.2",
    "polythene-mithril-ios-spinner": "^1.4.2",
    "polythene-mithril-list": "^1.4.2",
    "polythene-mithril-list-tile": "^1.4.2",
    "polythene-mithril-material-design-progress-spinner": "^1.4.2",
    "polythene-mithril-material-design-spinner": "^1.4.2",
    "polythene-mithril-menu": "^1.4.2",
    "polythene-mithril-notification": "^1.4.2",
    "polythene-mithril-radio-button": "^1.4.2",
    "polythene-mithril-radio-group": "^1.4.2",
    "polythene-mithril-raised-button": "^1.4.2",
    "polythene-mithril-ripple": "^1.4.2",
    "polythene-mithril-search": "^1.4.2",
    "polythene-mithril-shadow": "^1.4.2",
    "polythene-mithril-slider": "^1.4.2",
    "polythene-mithril-snackbar": "^1.4.2",
    "polythene-mithril-svg": "^1.4.2",
    "polythene-mithril-switch": "^1.4.2",
    "polythene-mithril-tabs": "^1.4.2",
    "polythene-mithril-textfield": "^1.4.2",
    "polythene-mithril-toolbar": "^1.4.2"
  },
  "devDependencies": {
    "polythene-core": "^1.4.2",
    "polythene-core-css": "^1.4.2"
  },
  "author": "Arthur Clemens <arthurclemens@gmail.com> (http://arthurclemens.com)",
  "homepage": "https://github.com/ArthurClemens/polythene",
  "license": "MIT"
}
