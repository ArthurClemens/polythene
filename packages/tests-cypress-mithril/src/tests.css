.page__test__component .align-items-center {
  display: flex;
  align-items: center;
}