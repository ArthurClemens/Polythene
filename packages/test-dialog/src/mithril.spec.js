import { runSnapshots } from "../../../scripts/tests/mithril-snapshots";
import { renderer, keys, Button, Dialog, DialogInstance } from "polythene-mithril";
import specTests from "./spec-tests.js";
import mithrilTests from "./tests-mithril.js";

runSnapshots({
  tests: specTests({ renderer, keys, Button, Dialog, DialogInstance }).concat(mithrilTests),
  renderer
});
