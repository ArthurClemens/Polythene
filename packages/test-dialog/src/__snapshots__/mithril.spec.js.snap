// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`No options - spec 1`] = `
"<div class=\\"pe-dialog\\">
  <div class=\\"pe-dialog__touch\\"></div>
  <div class=\\"pe-dialog__content\\">
    <div class=\\"pe-shadow pe-shadow--animated\\">
      <div class=\\"pe-shadow__bottom pe-shadow--z-3\\"></div>
      <div class=\\"pe-shadow__top pe-shadow--z-3\\"></div>
    </div>
    <form class=\\"pe-dialog-pane\\">
      <div class=\\"pe-dialog-pane__content\\">
        <div class=\\"pe-dialog-pane__body\\"></div>
      </div>
    </form>
  </div>
</div>
"
`;

exports[`Option: body - spec 1`] = `
"<div class=\\"pe-dialog\\">
  <div class=\\"pe-dialog__touch\\"></div>
  <div class=\\"pe-dialog__content\\">
    <div class=\\"pe-shadow pe-shadow--animated\\">
      <div class=\\"pe-shadow__bottom pe-shadow--z-3\\"></div>
      <div class=\\"pe-shadow__top pe-shadow--z-3\\"></div>
    </div>
    <form class=\\"pe-dialog-pane\\">
      <div class=\\"pe-dialog-pane__content\\">
        <div class=\\"pe-dialog-pane__body\\">
          Hello
        </div>
      </div>
    </form>
  </div>
</div>
"
`;

exports[`Option: class - spec 1`] = `
"<div class=\\"pe-dialog class-x\\">
  <div class=\\"pe-dialog__touch\\"></div>
  <div class=\\"pe-dialog__content\\">
    <div class=\\"pe-shadow pe-shadow--animated\\">
      <div class=\\"pe-shadow__bottom pe-shadow--z-3\\"></div>
      <div class=\\"pe-shadow__top pe-shadow--z-3\\"></div>
    </div>
    <form class=\\"pe-dialog-pane class-x\\">
      <div class=\\"pe-dialog-pane__content\\">
        <div class=\\"pe-dialog-pane__body\\"></div>
      </div>
    </form>
  </div>
</div>
"
`;

exports[`Option: className - spec 1`] = `
"<div class=\\"pe-dialog className-x\\">
  <div class=\\"pe-dialog__touch\\"></div>
  <div class=\\"pe-dialog__content\\">
    <div class=\\"pe-shadow pe-shadow--animated\\">
      <div class=\\"pe-shadow__bottom pe-shadow--z-3\\"></div>
      <div class=\\"pe-shadow__top pe-shadow--z-3\\"></div>
    </div>
    <form class=\\"pe-dialog-pane className-x\\">
      <div class=\\"pe-dialog-pane__content\\">
        <div class=\\"pe-dialog-pane__body\\"></div>
      </div>
    </form>
  </div>
</div>
"
`;

exports[`Option: element - spec 1`] = `
"<a class=\\"pe-dialog\\">
<div class=\\"pe-dialog__touch\\"></div>
<div class=\\"pe-dialog__content\\">
  <div class=\\"pe-shadow pe-shadow--animated\\">
    <div class=\\"pe-shadow__bottom pe-shadow--z-3\\"></div>
    <div class=\\"pe-shadow__top pe-shadow--z-3\\"></div>
  </div>
  <form class=\\"pe-dialog-pane\\">
    <div class=\\"pe-dialog-pane__content\\">
      <div class=\\"pe-dialog-pane__body\\"></div>
    </div>
  </form>
</div></a>
"
`;

exports[`Option: id - spec 1`] = `
"<div id=\\"id-x\\" class=\\"pe-dialog\\">
  <div class=\\"pe-dialog__touch\\"></div>
  <div class=\\"pe-dialog__content\\">
    <div class=\\"pe-shadow pe-shadow--animated\\">
      <div class=\\"pe-shadow__bottom pe-shadow--z-3\\"></div>
      <div class=\\"pe-shadow__top pe-shadow--z-3\\"></div>
    </div>
    <form class=\\"pe-dialog-pane\\">
      <div class=\\"pe-dialog-pane__content\\">
        <div class=\\"pe-dialog-pane__body\\"></div>
      </div>
    </form>
  </div>
</div>
"
`;

exports[`Option: modal with backdrop - spec 1`] = `
"<div class=\\"pe-dialog\\">
  <div class=\\"pe-dialog__backdrop\\"></div>
  <div class=\\"pe-dialog__touch\\"></div>
  <div class=\\"pe-dialog__content\\">
    <div class=\\"pe-shadow pe-shadow--animated\\">
      <div class=\\"pe-shadow__bottom pe-shadow--z-3\\"></div>
      <div class=\\"pe-shadow__top pe-shadow--z-3\\"></div>
    </div>
    <form class=
    \\"pe-dialog-pane pe-dialog-pane--header pe-dialog-pane--footer\\">
      <div class=\\"pe-dialog-pane__content\\">
        <div class=
        \\"pe-dialog-pane__header pe-dialog-pane__header--title\\">
          <div class=\\"pe-dialog-pane__title\\">
            Long dialog with a very long title that surely won't
            fit here
          </div>
        </div>
        <div class=\\"pe-dialog-pane__body\\">
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
        </div>
        <div class=
        \\"pe-dialog-pane__footer pe-dialog-pane__footer--buttons pe-button-row\\">
        <div class=\\"pe-dialog-pane__actions\\">
            <a class=\\"pe-button pe-text-button\\" tabindex=\\"0\\">
            <div class=\\"pe-button__content\\">
              <div class=\\"pe-ripple\\"></div>
              <div class=\\"pe-button__wash\\"></div>
              <div class=\\"pe-button__focus\\"></div>
              <div class=\\"pe-button__label\\">
                Cancel
              </div>
            </div></a><a class=\\"pe-button pe-text-button\\" tabindex=
            \\"0\\">
            <div class=\\"pe-button__content\\">
              <div class=\\"pe-ripple\\"></div>
              <div class=\\"pe-button__wash\\"></div>
              <div class=\\"pe-button__focus\\"></div>
              <div class=\\"pe-button__label\\">
                Save
              </div>
            </div></a>
          </div>
        </div>
      </div>
    </form>
  </div>
</div>
"
`;

exports[`Option: title and body (long text) - spec 1`] = `
"<div class=\\"pe-dialog\\">
  <div class=\\"pe-dialog__touch\\"></div>
  <div class=\\"pe-dialog__content\\">
    <div class=\\"pe-shadow pe-shadow--animated\\">
      <div class=\\"pe-shadow__bottom pe-shadow--z-3\\"></div>
      <div class=\\"pe-shadow__top pe-shadow--z-3\\"></div>
    </div>
    <form class=\\"pe-dialog-pane pe-dialog-pane--header\\">
      <div class=\\"pe-dialog-pane__content\\">
        <div class=
        \\"pe-dialog-pane__header pe-dialog-pane__header--title\\">
          <div class=\\"pe-dialog-pane__title\\">
            Long dialog with a very long title that surely won't
            fit here
          </div>
        </div>
        <div class=\\"pe-dialog-pane__body\\">
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
        </div>
      </div>
    </form>
  </div>
</div>
"
`;

exports[`Option: z (0) - spec 1`] = `
"<div class=\\"pe-dialog\\">
  <div class=\\"pe-dialog__touch\\"></div>
  <div class=\\"pe-dialog__content\\">
    <div class=\\"pe-shadow pe-shadow--animated\\">
      <div class=\\"pe-shadow__bottom pe-shadow--z-0\\"></div>
      <div class=\\"pe-shadow__top pe-shadow--z-0\\"></div>
    </div>
    <form class=\\"pe-dialog-pane\\">
      <div class=\\"pe-dialog-pane__content\\">
        <div class=\\"pe-dialog-pane__body\\">
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
        </div>
      </div>
    </form>
  </div>
</div>
"
`;

exports[`Option: z (5) - spec 1`] = `
"<div class=\\"pe-dialog\\">
  <div class=\\"pe-dialog__touch\\"></div>
  <div class=\\"pe-dialog__content\\">
    <div class=\\"pe-shadow pe-shadow--animated\\">
      <div class=\\"pe-shadow__bottom pe-shadow--z-5\\"></div>
      <div class=\\"pe-shadow__top pe-shadow--z-5\\"></div>
    </div>
    <form class=\\"pe-dialog-pane\\">
      <div class=\\"pe-dialog-pane__content\\">
        <div class=\\"pe-dialog-pane__body\\">
          <p>
            Lorem ipsum dolor sit amet, consectetur adipisicing
            elit, sed do eiusmod tempor incididunt ut labore et
            dolore magna aliqua. Ut enim ad minim veniam, quis
            nostrud exercitation ullamco laboris nisi ut aliquip ex
            ea commodo consequat. Duis aute irure dolor in
            reprehenderit in voluptate velit esse cillum dolore eu
            fugiat nulla pariatur. Excepteur sint occaecat
            cupidatat non proident, sunt in culpa qui officia
            deserunt mollit anim id est laborum.
          </p>
        </div>
      </div>
    </form>
  </div>
</div>
"
`;
